### PR TITLE
Add make lint

### PR DESCRIPTION
**Pull request by xlgao-zju**
cc @feiskyer 

Signed-off-by: Xianglin Gao <xlgao@zju.edu.cn>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -48,11 +48,23 @@ install: check-gopath
 uninstall:
 	rm -f $(BINDIR)/critest
 
+lint:
+	./hack/repo-infra/verify/go-tools/verify-gometalinter.sh
+
+install.tools: .install.gometalinter
+
+.install.gometalinter:
+	go get -u github.com/alecthomas/gometalinter
```
Comment (feiskyer) [inline Makefile]: why not make these two commands under `install.tools` directly?

Comment (xlgao-zju) [inline Makefile]: For adding other tools in the future. If you think we will not need any other tool, I'm fine with making these two commands under install.tools directly.

**Comment by xlgao-zju** [comment]
@feiskyer @mrunalp Updated and make the commands under install.tools directly

**Pull request merged**